Gemfile

```diff
@@ -27,6 +27,7 @@ gem "rubocop", "~> 0.54.0", require: false
 gem "sass-rails", "~> 5.0"
 gem "sidekiq"
 gem "sqlite3"
+gem "stripe-rails"
```

Comment:
tại sao lại cần gem này ?

Reply:
Gem này cung cấp thư viện giúp thanh toán thông qua stripe ạ